sass/main.scss

```diff
@@ -357,29 +357,63 @@ nav {
         }
     }
 
-    #mlh {
-        @include button;
-        @include button-effect($light-blue);
-        background-color: $light-blue;
+    #hackathon-buttons {
+        display: flex;
+        flex-direction: column;
         margin: 0 auto;
-        padding: 30px;
-        border-radius: 20px;
-
-        i, p {
-            display: inline;
-        }
-
-        p {
-            font-size: 1.2em;
-            font-weight: $font-medium;
+        align-items: flex-start;
+
+        #mlh {
+            @include button;
+            @include button-effect($light-blue);
+            background-color: $light-blue;
+            margin: 1.5em 0;
+            padding: 30px;
+            border-radius: 20px;
+    
+            i, p {
+                display: inline;
+            }
+    
+            p {
+                font-size: 1.2em;
+                font-weight: $font-medium;
+            }
+    
+            // TODO: Needs better alignment.
+            i.fa-arrow-right {
+                font-size: 1.6em;
+                color: $red;
+                vertical-align: text-bottom;
+                padding-right: 10px;
+            }
         }
-
-        // TODO: Needs better alignment.
-        i.fa-arrow-right {
-            font-size: 1.6em;
-            color: $red;
-            vertical-align: text-bottom;
-            padding-right: 10px;
+    
+        #gallery {
+            @include button;
+            @include button-effect($light-blue);
+            background-color: $light-blue;
+            opacity: 0.5;
```

Comment:
Idea: how about instead lowering opacity to show it's WIP, we use a diff icon from FontAwesome, and make it full opacity? 
<img width="1152" alt="Screen Shot 2020-12-16 at 7 19 22 PM" src="https://user-images.githubusercontent.com/2782749/102421928-9c6d7300-3fd3-11eb-9db5-bc0ef5612434.png">